digdag-core/src/main/java/io/digdag/core/database/migrate/Migration_20160926123456_AddDisabledColumnToSchedules.java

```diff
@@ -0,0 +1,13 @@
+package io.digdag.core.database.migrate;
+
+import org.skife.jdbi.v2.Handle;
+
+public class Migration_20160926123456_AddDisabledColumnToSchedules
+        implements Migration
+{
+    @Override
+    public void migrate(Handle handle, MigrationContext context)
+    {
+        handle.update("alter table schedules add column disabled_at bigint");
```

Comment:
An index needs update to keep `lockReadySchedules` fast even if there are many disabled schedules:

```
        if (context.isPostgres()) {
            handle.update("drop index schedules_on_next_run_time");
            handle.update("create index schedules_on_next_run_time where deleted_at is null");
        }
```